### ISSUE TITLE
Update check preferences ignored

### ISSUE THREAD

**Issue by SlimeDog** [Mr-EmPee/MysticalBarriers#12]
MysticalBarriers-1.8

Notifier in `notifier.yml` is configured `false`
```
notifier:
  # It's recommended to leave enabled this service! (If you disable it you are going to miss even security updates)
  # Before disabling it consider changing the report level
  enabled: false
  # The report-level declare which notification type you are going to receive.
  # Possible values: UPDATE, BUG_FIX, IMPORTANT
  #
  # The UPDATE report level warns you when is available an update that
  #  add new content or improve performance
  #
  # The BUG_FIX report level warns you when is available an update that fixes a bug
  #
  # The IMPORTANT report level warns you when is available an update that
  #  is HIGHLY recommended to download
  report-level: UPDATE
```
but every two hours, the log reports
```
[07:48:47] [Craft Scheduler Thread - 3/INFO]: [MysticalBarriers] Checking for updates...
[07:48:47] [Craft Scheduler Thread - 3/INFO]: [MysticalBarriers] Failed to check for updates: Connection timed out
[09:48:52] [Craft Scheduler Thread - 328/INFO]: [MysticalBarriers] Checking for updates...
[09:48:52] [Craft Scheduler Thread - 328/INFO]: [MysticalBarriers] Failed to check for updates: Connection timed out
```
(these are blocked at the network level).

**Comment by Mr-EmPee** [comment]
Fixed in 5000e9f2841bb23945cd2150506e526fc79990d1

**Comment by SlimeDog** [comment]
Confirmed fixed in 1.8.1. Thanks.
